.vsts-ci.yml

```
@@ -52,20 +52,20 @@ extends:
     template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
   parameters:
     containers:
-      alpine315WithNode:
-        image: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.15-WithNode
+      alpine319WithNode:
+        image: mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.19-WithNode
       cblMariner20Fpm:
         image: mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-fpm
-      centosStream8:
-        image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8
-      debianStretch:
-        image: mcr.microsoft.com/dotnet-buildtools/prereqs:debian-stretch
-      fedora36:
-        image: mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-36
+      CentOSStream9:
```

```suggestion
      centOSStream9:
```

References will need to be updated as well.